justfile

```diff
@@ -1,9 +1,12 @@
-
+CARGO_WORKSPACE := `dirname $(cargo locate-project --workspace --message-format=plain)`
+export ASPD_EXEC := CARGO_WORKSPACE / "target" / "debug" / "aspd"
+export BARK_EXEC := CARGO_WORKSPACE / "target" / "debug" / "bark"
```

Comment:
Hmm, I don't use the `./target` as the target dir. Maybe we can use the output of some cargo command to get the current target dir.

ChatGPT suggests:
```
$ cargo metadata --format-version 1 --no-deps | jq -r '.target_directory'
```
Which for me works. 

I have a global target dir at `/var/cargo-target`, f.e. Otherwise I have several multi-gigabyte target dirs floating around everywhere..